plugin-template

```diff
@@ -450,12 +446,20 @@ def write_template_section(config, name, plugin_root_dir, verbose=False):
         if relative_path.endswith(".j2"):
             template = env.get_template(relative_path)
             destination = destination_relative_path[: -len(".j2")]
-            write_template_to_file(
-                template,
-                plugin_root_dir,
-                destination,
-                template_vars,
-            )
+            if destination.startswith("pyproject.toml."):
+                utils.merge_toml(
+                    template,
+                    plugin_root_dir,
+                    destination,
+                    template_vars,
+                )
+            else:
+                template_to_file(
+                    template,
+                    plugin_root_dir,
+                    destination,
+                    template_vars,
+                )
```

Comment:
This (and `templates/github/pyproject.toml.tool.j2`)  means we are managing only the tools section, right?